ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -38,6 +41,40 @@ public ReactTextShadowNode()
         }
 
         /// <summary>
+        /// Sets the TextDecorationLine for the node.
+        /// </summary>
+        /// <param name="textDecorationLineValue">The TextDecorationLine value.</param>
```

Comment:
This is a case where I wish we could leave the <param> tag empty since the argument name is self-documenting.